warden-roboto/build.gradle.kts

```diff
@@ -108,11 +109,11 @@ publishing {
     publications {
         register("mavenJava", MavenPublication::class) {
             from(components["java"])
-            artifact(sourcesJar.get())
-            artifact(javadocJar.get())
+            if (this.name != "relocation") artifact(sourcesJar.get())
+            if (this.name != "relocation") artifact(javadocJar.get())
             pom {
-                name.set("Android Attestation")
-                description.set("Server-Side Android attestation library")
+                name.set("WARDEN-roboto")
+                description.set("Server-Side Android Attestation Library")
                 url.set("https://github.com/a-sit-plus/android-attestation")
```

Comment:
Is the url "https://github.com/a-sit-plus/android-attestation" on purpose?

Reply:
nice catch!